client/landing/stepper/declarative-flow/internals/steps.tsx

```diff
@@ -210,4 +210,14 @@ export const STEPS = {
 		slug: 'site-migration-instructions',
 		asyncComponent: () => import( './steps-repository/site-migration-instructions' ),
 	},
+
+	SITE_MIGRATION_SOURCE: {
+		slug: 'site-migration-import-from',
```

Comment:
I think we don't need to repeat 'site-migration'

Reply:
I would like to keep the `site-migration` prefix to help differentiate the steps from the steps in the existing Import flow.